lib/rules/media-feature-range-notation/__tests__/index.mjs

```diff
@@ -233,3 +313,44 @@ testRule({
 		},
 	],
 });
+
+testRule({
+	ruleName,
+	config: ['prefix', { except: ['exact-value'] }],
+
+	accept: [
+		{
+			code: '@media (width = 1px) {}',
+		},
+		{
+			code: '@media (1px = width) {}',
+		},
+		{
+			code: '@media (min-width: 1px) {}',
+		},
+		{
+			code: '@media (pointer: fine) {}',
+		},
+		{
+			code: '@media (color) {}',
+		},
+	],
+	reject: [
+		{
+			code: '@media (width > 1px) {}',
+			message: messages.expected('prefix'),
+			line: 1,
+			column: 8,
+			endLine: 1,
+			endColumn: 21,
+		},
+		{
+			code: '@media (width: 1px) {}',
+			message: messages.expected('context'),
+			line: 1,
+			column: 8,
+			endLine: 1,
+			endColumn: 20,
```

Comment:
Let's add `fix` to this using the context fixer.

Reply:
Added